sdk/template/Azure.Template/src/autorest.md

```diff
@@ -1,8 +1,17 @@
-# Generated code configuration
+# Azure SDK Code Generation for Data Plane
 
 Run `dotnet build /t:GenerateCode` to generate code.
 
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
 ``` yaml
 input-file:
-    -  $(this-folder)/swagger/mini-secrets.json
+- $(this-folder)/swagger/mini-secrets.json
+namespace: Azure.Template
+public-clients: true
```

Comment:
We should not use `public-clients: true` in a template -- this is not a recommended approach for data plane.